StencilFrontend.md

```diff
@@ -69,7 +68,7 @@ open Package.swift
 ```
 3. Change the target for Application to include "KituraStencil":
 ```swift
-.target(name: "Application", dependencies: [ "Kitura", "Configuration", "CloudEnvironment", "SwiftMetrics", "Health", "KituraStencil"]),
+.target(name: "Application", dependencies: [ "Kitura", "CloudEnvironment", "SwiftMetrics", "Health", "KituraStencil"]),
```

Comment:
This needs to include `SwiftKueryORM` and `SwiftKueryPostgreSQL` as well, from earlier.